source/reference/mongosync.txt

```diff
@@ -182,6 +193,27 @@ show an empty ``system.views`` collection in that database. The empty
 ``system.views`` collection will not change the accuracy of user
 data on the destination.
 
+Write Blocking
+~~~~~~~~~~~~~~
+
+By default, write-blocking is not enabled during synchronization. When
+write-blocking is enabled, ``mongosync`` blocks writes:
+
+- On the destination cluster during replication
+- On the source cluster while committing
+
+To enable write-blocking, start ``mongosync`` with
+``enableUserWriteBlocking`` set to ``true``.
+
+:ref:`Reverse synchronization <c2c-api-reverse>` requires
+write-blocking. To enable reverse synchronization between the source
+and destination clusters, start ``mongosync`` with ``reversible`` and
+``enableUserWriteBlocking`` set to ``true``.
```

Comment:
I would also mention that once `reversible` / `enableUserWriteBlocking` is set, they are not allowed to change. And users are only able to config them the value of these two options at the time of `/start` (e.g. not at the time of`/reverse`).